HACKING.md

```diff
@@ -0,0 +1,17 @@
+This is a plugin for cockpit.
+Install cockpit according to https://cockpit-project.org/running.html
+
+Pull the repository and compile
+
+```sh
+$ git clone https://github.com/skobyda/cockpit-certificates.git
+$ cd cockpit-certificates
+$ make
+```
+
+This project is based on cockpit's [starter-kit](https://github.com/cockpit-project/starter-kit)
+For development, run your module straight out of the git tree:
+```sh
+# mkdir -p ~/.local/share/cockpit
+# ln -s `pwd`/dist ~/.local/share/cockpit/certificates
```

Comment:
Please use a consistent shell prompt. This looks like it would need to run as root, which is not only unnecessary, but actively doesn't work. Also, there is `make devel-install` for this.

Reply:
Right, how does one make a starter-kit based plugin visible when user is logged into cockpit as root?

Reply:
That's not a common use case -- logging in as root should really not be a thing. But then of course you'd run these as root.